cg/store/models.py

```diff
@@ -1051,6 +1051,19 @@ class IlluminaSequencingRun(InstrumentRun):
 
     __mapper_args__ = {"polymorphic_identity": DeviceType.ILLUMINA}
 
+    @hybrid_property
```

Comment:
What are these decorators? Never saw them before

Reply:
`flow_cell` is not an attribute of the model, so this decorator allows SQLAlchemy to recognize `flow_cell` as an attribute

Reply:
replaced for a simpler solution